src/pages/Projects/styles/Projects.styled.ts

```diff
@@ -32,8 +29,15 @@ export const Image = styled.img`
   width: 15%;
 `
 
-export const Divider = styled.div`
+export const DividerContainer = styled.div`
   height: 1px;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+`
+
+export const Divider = styled.div`
+  height: 100%;
```

Comment:
try to avoid %. if possible, use vw and vh.

Rationale: % takes a reference to its closest parent. If the component gets messier, there may undesired side visual effects.

Reply:
replaced some to vw and vh where possible in commit 9ec412c, thanks!